src/services/qrcode/__tests__/index.test.unit.ts

```diff
@@ -0,0 +1,18 @@
+import { createHmac } from 'crypto';
+import * as QRCode from '..';
+import { getConfig } from '../../../../config';
+
+
+describe('QRCode Service', () => {
+  const config = getConfig(process.env.NODE_ENV);
+
+  describe('create', () => {
```

Comment:
Do we need a fail case here?

Reply:
I don't think I can force a failure here (without annoying the compiler). I'm just calling 3rd party functions.